### PR TITLE
Remove admin namespace phase i

### PR DESCRIPTION
**Pull request by tchorzewski1991**
I do not remove namespece yet. Routes are just commented. This is because some I suppose some elements from controllers etc. could be quite useful.

For now:

- [x] navbar and user popover menu
- [x] user login/sign-up/edit-accout  ability and views
- [x] creating new records associated with user and price

I removed temporary Devise::Invitable module. I will add it after this.

### CODE REVIEW

File: app/controllers/records_controller.rb [hunk 1]
```diff
@@ -28,8 +28,7 @@ def create
         artist: @price.artist,
         label: @price.label,
         record_format: @price.record_format,
-        #temporary hardcoded
-        user: User.last })
```
Comment (holden) [inline app/controllers/records_controller.rb]: Good god! ;-)

File: app/views/devise/invitations/new.html.slim [hunk 2]
```diff
@@ -1,11 +0,0 @@
-#invitation
-  = simple_form_for resource, :as => resource_name, :url => invitation_path(resource_name), :html => {:method => :post} do |f|
-    = devise_error_messages!
-    .panel.panel-default
-      .panel-heading
```
Comment (holden) [inline app/views/devise/invitations/new.html.slim]: I'd forgotten how much I hate slim!

File: config/routes.rb [hunk 3]
```diff
@@ -1,11 +1,23 @@
...
-  devise_scope :user do
-    get "login", to: "devise/sessions#new"
+  devise_for :users,
+  path: '',
+  path_names: {
+    sign_in: 'login',
+    sign_out: 'logout'
+    },
+  controllers: {
+    registrations: "users/registrations",
```
Comment (holden) [inline config/routes.rb]: Disable this for now.

File: app/controllers/records_controller.rb (hunk 1 above)
Comment (tchorzewski1991) [inline app/controllers/records_controller.rb]: This is no more hardcoded. It was before I create ability to sign in user.

**Pull request merged**